test/root/root_v2_api_test.rb

```diff
@@ -25,8 +25,8 @@ def app
   end
 
   def test_features
-    proc2 = lambda{ ['a', 'b']}
-    proc3 = lambda{raise "Should not be called"}
+    proc2 = -> { ['a', 'b'] }
+    proc3 = -> {raise "Should not be called" }
```

Comment:
This whitespacing looks inconsistent.

Reply:
aw, I fixed up the `proc2` line manually. Now also `proc3`.

Reply:
I guess, there would be another (currently disabled) cop also for that.